<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoalManager</class>
 <widget class="QWidget" name="GoalManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Goal Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">GoalManager {
    background-color: #e8f5e9;
}
QGroupBox {
    background-color: white;
    border: 1px solid #c8e6c9;
    border-radius: 8px;
    margin-top: 10px;
    padding: 10px;
    font-weight: bold;
}
QLineEdit, QDoubleSpinBox, QDateEdit, QComboBox {
    background-color: #f9fff9;
    border: 1px solid #a5d6a7;
    border-radius: 4px;
    padding: 4px;
        color: black;
 }
QPushButton {
   background-color: #43a047;
   color: white;
   font-weight: bold;
   border-radius: 6px;
   padding: 6px 10px;
}
QPushButton:hover {
   background-color: #2e7d32;
}
QListWidget {
  background-color: white;
  border: 1px solid #c8e6c9;
  border-radius: 8px;
}

QLabel {
        color: black;
}

QCalendarWidget {
   color: black;
    border: 1px solid #81c784;
    border-radius: 8px;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QPushButton" name="homeBtn">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #43a047;
    color: white;
    font-size: 20px;
    border-radius: 6px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #2e7d32;
}</string>
       </property>
       <property name="text">
        <string>üè†</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #e8f5e9;</string>
       </property>
       <property name="text">
        <string>üèÜ Your Financial Goals</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="goalDetailsGroup">
     <property name="styleSheet">
      <string notr="true">color:black;
</string>
     </property>
     <property name="title">
      <string>Goal Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="goalNameInput"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="targetLabel">
        <property name="text">
         <string>Target Amount:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="goalTargetInput">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="currentLabel">
        <property name="text">
         <string>Current Amount:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="goalCurrentInput">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="startLabel">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="goalStartInput">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="endLabel">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDateEdit" name="goalEndInput">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addGoalBtn">
       <property name="text">
        <string>‚ûï Add Goal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateGoalBtn">
       <property name="text">
        <string>üîÑ Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeGoalBtn">
       <property name="text">
        <string>üóë Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearFieldsBtn">
       <property name="text">
        <string>üßπ Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="goalListWidget">
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
